tests/infra/network.py

```diff
@@ -433,13 +433,16 @@ def start(self, args, **kwargs):
         mc = max(1, args.initial_member_count)
         initial_members_info = []
         for i in range(mc):
+            member_data = None
+            if i < args.initial_trusted_authority_count:
+                member_data = {"is_trusted_authority": True}
+            elif i < args.initial_trusted_authority_count + args.initial_operator_count:
+                member_data = {"is_operator": True}
```

Comment:
Do we have a check somewhere that `mc > (initial_trusted_authority_count + initial_operator_count)` ?

Reply:
We do as of https://github.com/microsoft/CCF/pull/4343/commits/f2cae12a91725da39d1a4d0d60ea6bb7fe45f6ef